Support/bin/gltf_export_textures.py

```diff
@@ -74,7 +74,20 @@ def process_request(request):
     if not os.path.isdir(outdir):
         os.makedirs(outdir)
 
-    im.save(request["destination"])
+    # If the image is a jpeg then strip off the alpha channel
+    if (
+        request["destination"].endswith(".jpg")
+        or request["destination"].endswith(".jpeg")
+    ) and im.mode == "RGBA":
+        im = im.convert("RGB")
+
+    # Catch OSError and re-raise with a more informative message
+    try:
+        im.save(request["destination"])
+    except OSError as e:
+        raise OSError(
```

Comment:
pylint has two lint errors here. The one on line 87... harmless but legitimate. The one of this line is a good suggestion